dev-vcs/tig/tig-2.2.recipe

```diff
@@ -4,13 +4,13 @@ Tig functions mainly as a git repository browser. It can also help in the \
 staging changes for commit at chunk level and act as a pager for output \
 different git commands."
 HOMEPAGE="http://jonas.nitro.dk/tig/"
```

Comment:
Hmm, it would be nice to have both URLs in HOMEPAGE, like this:
```
HOMEPAGE="http://jonas.nitro.dk/tig/
	https://jonas.github.io/tig/"
```
Reminder: the second line starts with a single tab.

(I'm quite sure it is OK to have more than one address in HOMEPAGE; I've seen a few recently, although I don't remember in which recipes it was.)